pages/thanks-driver.html

```diff
@@ -9,20 +9,12 @@
     <div class="wrapper single-column offset-top">
         <div class="bannerbox">
             <div class="bannerbox__content">
-                <h1 id="thanks-header">Thank you</h1>
```

Comment:
Can we keep the id for the h1 element until after Nov 7, please? The automated tests use it.

Reply:
Sure, let's update the tests when we merge after Nov 7.